Lec/rigraid_connacht_tar_eis_aillela_agus_meadba.trig

```diff
@@ -17,24 +17,263 @@
         dcterms:format "application/trig" ;
         prov:asDerivedFrom <https://www.isos.dias.ie/libraries/RIA/RIA_MS_23_P_2/small_jpgs/353.jpg>, <https://www.isos.dias.ie/libraries/RIA/RIA_MS_23_P_2/large_jpgs/354.jpg>, <https://www.isos.dias.ie/libraries/RIA/RIA_MS_23_P_2/small_jpgs/353.jpg>, <https://www.isos.dias.ie/libraries/RIA/RIA_MS_23_P_2/large_jpgs/354.jpg> .
 
+<#Aillella>
+    a irishTitles:Rí;
+    irishRel:genName "Aillella";
+    irishRel:nomName "Ailill";
+    irishTitles:succeededBy <#MaineAithremail>;
+    owl:sameAs <http://example.com/Rawl_B512/ferchuitred_medba.trig#Ailella>.
+
+<#Meadba>
+    a irishTitles:Rí;
+    foaf:gender "female";
+    irishRel:genName "Meadba";
+    irishRel:nomName "Medb";
+    irishTitles:succeededBy <#MaineAithremail>;
+    owl:sameAs <http://example.com/Rawl_B512/ferchuitred_medba.trig#MedbCruachna>.
+
+<#MaineAithremail>
+    a irishTitles:Rí;
+    irishRel:nomName "Maine Aithremail";
+    irishTitles:succeededBy <#Shanb>;
+    owl:sameAs <#FelimMaineAithreamail>.
+
+<#Shanb>
+    a irishTitles:Rí;
+    irishTitles:reignLength "26";
```

Comment:
This should be 26 without the quotes.  It is defined in the ontology as a nonNegativeInteger so it should be without quotes.

Reply:
Done